<launch>
  <node pkg="opti_track" name="opti_track_node" type="opti_track_node" output="screen">
    <param name="server_address" value="192.168.1.13" />
    <param name="client_address" value="192.168.1.23" />
  </node>
</launch>
